pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -27,16 +27,31 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/log"
 
-	"kubevirt.io/kubevirt/pkg/network/cache"
-	netsetup "kubevirt.io/kubevirt/pkg/network/setup"
 	netvmispec "kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, converterContext *converter.ConverterContext, dom cli.VirDomain, domain *api.Domain) error {
-	vmConfigurator := netsetup.NewVMNetworkConfigurator(vmi, cache.CacheCreator{})
+type vmConfigurator interface {
```

Comment:
IMO this name is too generic for an interface that sets up the network.
Please note that it resides in a package (virtwrap) that does more than networking.

How about `podNetworkConfigurator` or something along those lines?

Reply:
the only method this interface provides is named `SetupPodNetworkPhase2` . Thus, IMHO it's obvious it configures network related stuff.

`podNetworkConfigurator.SetupPodNetworkPhase2` stutters a bit, and I'd like to avoid that if possible.

Reply:
> the only method this interface provides is named SetupPodNetworkPhase2 . Thus, IMHO it's obvious it configures network related stuff.

I agree that its behavior is clear from looking at its methods, but not so much from looking at its name.

`podNetworkConfigurator.SetupNetworkPhase2` or `podNetworkConfigurator.SetupPhase2` is long, but clear.

Please consider it.

Reply:
Fair enough.

Reply:
Thank you.

If I were the only user of this method, I would have changed it, fwiw.